shell/browser/api/electron_api_desktop_capturer.cc

```diff
@@ -122,6 +125,11 @@ void DesktopCapturer::OnSourceUnchanged(DesktopMediaList* list) {
   UpdateSourcesList(list);
 }
 
+void DesktopCapturer::SetSkipCursor(const std::string& device_id,
+                                    bool skip_cursor) {
+  content::MediaDeviceRequest::SetSkipCursor(device_id, skip_cursor);
```

Comment:
since this is a static method, which does not need a `DesktopCapturer` object for context, it should be exposed directly on the bindings object in `Initialize`.

Reply:
Done. Removed the method from DesktopCapturer and moved it to Initialize